client/ReviewItem.jsx

```diff
@@ -1,13 +1,33 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import { Container, Row, Col } from 'reactstrap';
+import VoteButton from './VoteButton.jsx';
 
 const ReviewItem = props => (
-  <tr>
-    <td>Stars: {props.review.stars}<br />
-    Review: {props.review.text}<br />
-    Useful: {props.review.useful}&nbsp;Funny: {props.review.funny}&nbsp;Cool: {props.review.cool}
-    </td>
-  </tr>
+  <div className="review-wrapper">
+    <div className="review-content">
+      Stars: {props.review.stars}
+      <br />
+      {props.review.text.map(line => <Line line={line} />)}
```

Comment:
I think you need to add a key when you map through a list. You can use the _id or another way to assign to a key like key = {...} after line ={line}

Reply:
I will add a key.   Thanks.